include/crocoddyl/core/action-base.hxx

```diff
@@ -48,6 +48,12 @@ void ActionModelAbstractTpl<Scalar>::quasiStatic(const boost::shared_ptr<ActionD
     throw_pretty("Invalid argument: "
                  << "x has wrong dimension (it should be " + std::to_string(state_->get_nx()) + ")");
   }
+#ifndef NDEBUG
```

Comment:
You could remove this `ifndef` and simply do assert of x.tail(state_->get_nv()).isZero(). That would be simpler.